ros2bag/test/resources/incomplete_qos_duration.yaml

```diff
@@ -0,0 +1,7 @@
+/test_topic:
+  history: keep_all
+  depth: 0
+  reliability: reliable
+  durability: volatile
+  deadline:
+    sec: 2147483647
```

Comment:
what is the significance of this value?
Can you insert a yaml comment describing why it was chosen?

Reply:
`ULONG_MAX`. No significance tho.